checkfiles/checkfiles.py

```diff
@@ -278,7 +278,9 @@ def process_old_illumina_read_name_pattern(read_name,
             if (flowcell.find('-') != -1 or
                flowcell.find('_') != -1):
                 flowcell = 'TEMP'
-            lane_number = arr[1]
+            lane_number = '0'
+            if arr[1].isdigit():
+                lane_number = arr[1]
```

Comment:
added comments, not sure I am listing all cases, but I am explaining this specific choice to put zero in lane number